src/batou_ext/geoip.py

```diff
@@ -16,9 +16,14 @@ class GeoIPDatabase(batou.component.Component):
 
     Needs curl and gunzip inside $PATH
     """
+    license_key = None
+    url = batou.component.Attribute(
+        str,
+        "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&suffix=tar.gz&license_key={{component.license_key}}")
 
     def configure(self):
 
+        self.download_url = self.expand(self.url)
```

Comment:
I think that's not necessary, the Attribute defaults to expanding anyway.

Reply:
Removed